doc/src/guides/solving/solve-ode.md

```diff
@@ -0,0 +1,262 @@
+# Solve an Ordinary Differential Equation Algebraically
+
+Use SymPy to solve an ordinary differential equation algebraically. For example,
+solving $y''(x) + 9y(x)=0 $ yields $ y(x)=C_{1} \sin(3x)+ C_{2} \cos(3x)$.
+
+Alternatives to consider:
+- *which SciPy functions? from
+  https://docs.scipy.org/doc/scipy/reference/integrate.html?highlight=ode?*
```

Comment:
I think solve_ivp is good for most instances https://docs.scipy.org/doc/scipy/reference/generated/scipy.integrate.solve_ivp.html#scipy.integrate.solve_ivp. Or we can just reference this page https://docs.scipy.org/doc/scipy/reference/integrate.html#solving-initial-value-problems-for-ode-systems